app/forms/contact_form.rb

```diff
@@ -0,0 +1,9 @@
+class ContactForm
+  include ActiveModel::Model
+
+  attr_accessor :want_learn, :want_tell, :email_or_tel, :comments
+
+  def submit(_)
+    valid?
```

Comment:
On second thought, we should keep the form object in case we decide to add validations later on. This will allow the code to embrace change, and will make it very clear to future developers how and where to add validations should they be needed. Here are relevant quotes from Sandi Metz's *Practical Object-Oriented Design in Ruby*:

> The structure of every class is a message to future maintainers of the application. It reveals your design intentions. For better or for worse, the patterns you establish today will be replicated forever.

> There is a chance that someone else will reuse [your class] , or create new code that follows its pattern while you are waiting for better information. Other developers believe that your intentions are reflected in the code; when the code lies you must be alert to programmers believing and then propagating that lie.

> You can arrange the code so that [your class] will be easy to change even if you don't know what changes will come. Because change is inevitable, coding in a changeable style has big future payoffs. As an additional bonus, coding in these styles will improve your code, today, at no extra cost.